code/mob/living/carbon/human/procs/get_desc.dm

```diff
@@ -410,6 +410,29 @@
 		items = copytext(items, 3)
 		. += "<br>[SPAN_NOTICE("[src] is juggling [items]!")]"
 
+	if (src.reagents.has_reagent("ethanol"))
```

Comment:
this does not appear to check for the trait on the viewer

Reply:
That's intentional-- someone else here mentioned that it made sense as more of an everyone thing, so I did away with locking it behind bartender training